org.civicrm.reqangular/src/common/components/heatmap.component.js

```diff
@@ -5,7 +5,7 @@ define([
   'common/moment',
   'common/modules/components'
 ], function (_, moment, components) {
-  return components.component('heatmap', {
```

Comment:
Could you please explain why this was removed?

Reply:
just the return, not the component definition. It was removed because returning the component is not standard. It also blocks you from doing things like this:

```js
return components.component({
   // ...
});

controller.$inject = ['$scope']; // <--- won't work because of the return.

function controller ($scope) {
  // ...
}
```